### PR TITLE
Added a repo Link For First Timers and repo link for Coding for Beginners

### PR DESCRIPTION
**Pull request by Samar1110**
Issue : #454 

## Contributing

If you want to add a project to this list, please make sure that:

- [x] You read the [contributing guide](https://github.com/OtacilioN/awesome-hacktoberfest-2020/blob/master/CONTRIBUTING.md)
- [x] The repository has hacktoberfest as topic and counts for hacktoberfest event
- [x] The project can't be just an "add your name" or simply + 1 contribution to hacktoberfest
- [x] Place them in alphabetical order
- [x] One repository per PR/contribution - _avoid conflicts and simplify review_
- [x] Your pull request is not spammy just adding an empty line, otherwise it will be marked as invalid and not count to hacktoberfest.

If they meet the requirements above, feel free to create a PR. Happy Hacking! :sparkles:

### CODE REVIEW

**Comment by Samar1110** [comment]
@OtacilioN please review and merge my PR for Hactoberfest 2022

File: README.md [hunk 1]
```diff
@@ -19,6 +19,7 @@
 
 ## Repositories Supporting First-Timers
 
+- [AliceWonderland](https://github.com/AliceWonderland/hacktoberfest)
```
Comment (OtacilioN) [inline README.md:22]: not valid

**Review by OtacilioN — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (Samar1110) [reply README.md]: Issue : #454 

Added a Valid Repo now and Removed Invalid Repo
@OtacilioN please review and Merge my PR for Hactoberfest-2022

**Comment by Samar1110** [comment]
Added a New Valid Repo and Removed the invalid repo
@OtacilioN please review and merge my PR for Hactoberfesr-2022

**Pull request closed without merging**